runtime/src/bank/partitioned_epoch_rewards/calculation.rs

```diff
@@ -668,10 +741,21 @@ mod tests {
     fn test_rewards_computation() {
         solana_logger::setup();
 
-        let expected_num_delegations = 100;
-        let bank = create_default_reward_bank(expected_num_delegations, SLOTS_PER_EPOCH)
-            .0
-            .bank;
+        // Delegations with sufficient stake to get rewards (2 SOL).
+        let delegations_with_rewards = 100;
```

Comment:
I was tempted to put 1_000_000 to match the numbers we're seeing on mainnet, but unfortunately, even 1_000 causes `create_reward_bank_with_specific_stakes` to execute for over a minute... I will try to fix that and improve the test in a separate PR.